### DISCUSSION TITLE
Adding Polkadot to Wormhole Ecosystem

### DISCUSSION THREAD

**Discussion by amalnathsathyan** [wormhole-foundation/wormhole#557] [Feature requests] [answered]
Along with current bridges in the ecosystem, it would be great if the wormhole can support Polkadot too.

The rise of Multichain projects demands a high need to have interoperability between the chains. Polkadot itself is a web of blockchains along with the parachains. Also, Polkadot is one of the top 10 crypto ecosystems out there. 
A bridge to Polkadot means a bridge to 100s of parachains also.

Currently, I am working on an NFT project that is multichain, multi tribe. We have some tribes living on Solana. The next one is launching soon on the Polkadot-Kusama networks. The RMRK framework of Polkadot is allowing us to mint NFTs that have some extra features including fragment-ability. We are also in a need to connect the Polkadot <> Solana, just like many projects out there. But to start and secure a bridge is a behemoth task for us. 

If anyone wants to see what we are building feel free to have a look at : https://metasafari.world/

**Comment by cyberk1337** [comment]
Great 👏👏

**Comment by justinschuldt** [comment]
Please see: https://github.com/certusone/wormhole/blob/dev.v2/CONTRIBUTING.md#can-you-add-random-blockchain

**Reply by amalnathsathyan** [reply]
Have seen this. But how can I convince the guardian nodes for that? I thought expressing a proposal here would be appropriate !

**Reply by leoluk** [reply]
The Wormhole community hangs out on Discord, best to reach out there and reference this issue: https://discord.gg/xsT8qrHAvV

In the end it's them you'll have to convince to support the new chain.

**Reply by amalnathsathyan** [reply]
Actually, they had advised me to express this issue here

**Comment by coderfengyun** [comment]
Any progress here ?

**Comment by evan-gray** [comment, marked as answer]
Wormhole launched on Karura and Acala [in June](https://twitter.com/wormholecrypto/status/1536364579047055360) 🙂
